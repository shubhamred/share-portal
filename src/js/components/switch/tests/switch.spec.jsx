import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Switch from '../switch';

configure({ adapter: new Adapter() });

describe('components/switch', () => {
  it('Renders default options', () => {
    const component = shallow(
      <Switch />
    );
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
