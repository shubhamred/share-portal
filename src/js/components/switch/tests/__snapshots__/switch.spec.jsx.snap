// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/switch Renders default options 1`] = `
<WithStyles(ForwardRef(Switch))
  checked={false}
  classes={
    Object {
      "checked": "makeStyles-checked-5",
      "root": "makeStyles-root-1",
      "switchBase": "makeStyles-switchBase-2",
      "thumb": "makeStyles-thumb-3",
      "track": "makeStyles-track-4",
    }
  }
  onChange={[Function]}
/>
`;
